content/appendices.md

```diff
@@ -611,11 +611,20 @@ characters permitted in user ID localparts. There are currently active
 users whose user IDs do not conform to the permitted character set, and
 a number of rooms whose history includes events with a `sender` which
 does not conform. In order to handle these rooms successfully, clients
-and servers MUST accept user IDs with localparts from the expanded
-character set:
+and servers MUST accept user IDs with localparts consisting of any legal
+unicode codepoint except for `:`, including zero characters. Localparts
+MUST be valid UTF-8 sequences.
+
+Servers SHOULD NOT produce user IDs with localparts outside of the following
```

Comment:
surely servers SHOULD NOT produce user IDs with localparts outside of `a-z`, `0-9`,
`.`, `_`, `=`, `-`, `/`, and `+` ?

Reply:
This part probably just needs some rewording, but it depends on how the above thread about the three user ID classes is resolved.